README.md

```diff
@@ -20,12 +20,11 @@ Packages in this repository are designed to work with the following browsers
 
 ### Internet Explorer Support
 
-Active support for Internet Explorer was dropped in August 2021. Some packages use `core-js` and `babel` to polyfill unsupported features, but this will be phased out over time.
+Active support for Internet Explorer was dropped in August 2021. As of 04/2021 we no longer output CommonJS and instead output ESM. We also do not provide any polyfills. These will need to be configured in your own environment.
```

Comment:
We can mention that `@availity/workflow` continues to provide polyfills, but if you are not using workflow then you'll need to polyfill appropriately